README.md

```diff
@@ -975,32 +975,38 @@ This will yield streamed output from the run like the following:
 ```
-
 ## How to work with Azure OpenAI
 
-For Azure OpenAI scenarios use the [Azure SDK](https://github.com/Azure/azure-sdk-for-net) and more specifically the [Azure OpenAI client library for .NET](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/openai/Azure.AI.OpenAI/README.md). 
+Switching from OpenAI to Azure OpenAI is simple, and in most cases requires little to no code changes. To get started quickly, check out the starter kit at https://aka.ms/openai/start. If you want to understand how endpoint switching works, you can also read: https://aka.ms/openai/switch.
+
+### Secure Access with Microsoft Entra ID (No API Keys)
+The starter kit includes examples showing how to call Azure OpenAI securely using Microsoft Entra ID instead of API keys. This is the recommended approach for production scenarios. Here’s a direct link to the .NET sample using Entra ID in the starter kit: https://github.com/Azure-Samples/azure-openai-starter/blob/main/src/dotnet/responses_example_entra.cs
 
-The Azure OpenAI client library for .NET is a companion to this library and all common capabilities between OpenAI and Azure OpenAI share the same scenario clients, methods, and request/response types. It is designed to make Azure specific scenarios straightforward, with extensions for Azure-specific concepts like Responsible AI content filter results and On Your Data integration.
+Below is the core pattern using the OpenAI SDK for .NET with Azure OpenAI + Entra ID:
 
 ```c#
-AzureOpenAIClient azureClient = new(
-    new Uri("https://your-azure-openai-resource.com"),
-    new DefaultAzureCredential());
-ChatClient chatClient = azureClient.GetChatClient("my-gpt-35-turbo-deployment");
+using Azure.Identity;
+using OpenAI;
+using OpenAI.Responses;
+using System.ClientModel.Primitives;
 
-ChatCompletion completion = chatClient.CompleteChat(
-    [
-        // System messages represent instructions or other guidance about how the assistant should behave
-        new SystemChatMessage("You are a helpful assistant that talks like a pirate."),
-        // User messages represent user input, whether historical or the most recen tinput
-        new UserChatMessage("Hi, can you help me?"),
-        // Assistant messages in a request represent conversation history for responses
-        new AssistantChatMessage("Arrr! Of course, me hearty! What can I do for ye?"),
-        new UserChatMessage("What's the best way to train a parrot?"),
-    ]);
-
-Console.WriteLine($"{completion.Role}: {completion.Content[0].Text}");
+var endpoint = Environment.GetEnvironmentVariable("AZURE_OPENAI_ENDPOINT")
+    ?? throw new InvalidOperationException("AZURE_OPENAI_ENDPOINT is required.");
+
+var client = new OpenAIResponseClient(
+    "gpt-5-mini",
+    new BearerTokenPolicy(new DefaultAzureCredential(), "https://cognitiveservices.azure.com/.default"),
```

Comment:
@trrwilson, I was under the impression the recommended scope now is ai.azure.com rather than cognitiveservices.azure.com... is that correct or am I misremembering?